agent/action/pipeline_manager.py

```
@@ -1,807 +1,464 @@
-
-            # 执行魔改
-            context.override_pipeline({target_node: patch_data})
-            utils.mfaalog.info(f"[Py] 🔧 节点 [{target_node}] 已打补丁")
-            return True
-        except Exception as e:
-            utils.mfaalog.error(f"[Py] PatchNode 失败: {e}")
-            return False
+# ------------------------------------------------------------------------------
+# 还原点账本
+# ------------------------------------------------------------------------------
+# 按 (tasker 实例, 任务号) 分桶：运行时改写本就随任务结束失效，账本必须同生共死，
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)